kmvpvl/plutchart
# Beta with matches (#72)

* api prepared

* fix: mongo error message made sync

* mongo added

* fix: return async call of connect

* fix: prevent database call if not tg_user_id received

* fix: reject requests to alian domains

* fix: if user delete its account user may be undefined

* fix: it was too earlier

* tool + landing

* fix: mongo minor settings

* fix: dockerignore

* check: if delay?

* test: all 200 response

* tool and landing are in prod mode

* move tg settings to special function

* robots and sitemap for plutchart.com

* tgsetting returns value to caller

* Change "My settings" bot function implementation from callback to command Fixes #19

* Kmvpvl/issue28 (#30)

landing language choosing changed

* check: collect logs for api

* fix: error with folders

* add remaining services

* check: health page

* check: JSON return trying

* fix: 500

* fix: health.health

* fixed: return test health data

* Landing: center content with left and right margins on too wide displays Fixes #32

* Tool: chapter for support зкщещ

* info about user by TG Id

* duplicate button

* new components created: Flower, Pending, Chart, Charts

* fix: onWheel prevent page scrolling

* fix: set access to public

* fix: slider

* complex emotions demo

* fix: support staff - count of assessments

* enh: add emotion vectors to support staff

* fix: delete source map to disappear warnings

* enh: eslint + prettier

* prepared to publish components

* pending replaced

* fix: chart component updated

* dump ballast: nginx, certbot, wiki

* fix: delete remaining files

* wrong default language for chart and charts components Fixes #56

* fix: 0.1 by every wheel event because safari and chrome send different values

* Implement proposal to describe emotional vector Fixes #53

* logo-container min-height: 100% wrong for Firefox formatting Fixes #60

* common components updated

* toast style fixed

* commons to external repository

* fix: new features

* users rating list

* wiki engine to landscape Fixes #67

* united docker in one

* prepare to match implementation

* chat with mutuals

---------
